From c75953c2010b9932bd921096523e190cbaf3b578 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Tue, 28 Oct 2014 22:32:18 +0200
Subject: [PATCH] stop unit_file_preset_all doing stupid things

populating shitload of useless symlinks into /storage/.config/system.d
is not so-nice fuckery. and systemctl preset-all is useless on an
embedded distro like dekockblock anyway.
---
 src/shared/install.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/src/shared/install.c b/src/shared/install.c
index 035b44c..74aa7c5 100644
--- a/src/shared/install.c
+++ b/src/shared/install.c
@@ -2043,6 +2043,11 @@ int unit_file_preset_all(
                 UnitFileChange **changes,
                 unsigned *n_changes) {
 
+        // meh systemctl preset-all.
+        // I DONT WANT shitload of symlink populated
+        // just because systemd "thinks" (o_O) /etc is empty
+        return 0;
+
         _cleanup_(install_context_done) InstallContext plus = {}, minus = {};
         _cleanup_lookup_paths_free_ LookupPaths paths = {};
         _cleanup_free_ char *config_path = NULL;
-- 
1.7.2.5

